cmd/gaia/app/app.go

```diff
@@ -140,11 +140,17 @@ func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest b
 		&stakingKeeper, app.paramsKeeper.Subspace(slashing.DefaultParamspace),
 		slashing.DefaultCodespace,
 	)
+
+	govRouter := gov.NewRouter()
+	govRouter.AddRoute(gov.RouterKey, gov.ProposalHandler).
+		AddRoute(params.RouterKey, params.NewProposalHandler(app.paramsKeeper))
```

Comment:
k - so I mentioned a similar comment surrounding the use of `init` within `x/params` in order to register the codec. etc with gov. I think those lines from `init` should be moved right here OR we should consider changing `NewProposalHandler` to maybe `NewParamChangeProposalHandler` - just because it should be absolutely clear looking at the code in `app.go` that we're routing param change proposal stuff into governance. Right now it's not really obvious without digging around a little bit.

Reply:
A fair point of criticism, but see my reasoning and response above in the initial comment. I think it's ultimately best to keep the `init` logic local to implementation/definition.

Reply:
k well if we're keeping it here we should rename`NewProposalHandler` to `NewParamChangeProposalHandler` or something more descriptive

Reply:
Agreed. Will update.